pom.xml

```diff
@@ -902,6 +908,29 @@
             <autoVersionSubmodules>true</autoVersionSubmodules>
           </configuration>
         </plugin>
+        <plugin>
```

Comment:
The parent POM doesn't actually declare dependencies (aside from `maven-enforcer-plugin`). This defines a common set of configuration which other modules can include. 

For now I've added it only for `java-sdk-all`. We may want to add it for other modules down the line, but I'm not sure about that decision yet-- I imagine the set of validations we enforce for SDKs would be stricter than runners or example code.